hera_pspec/uvpspec.py

```diff
@@ -78,7 +80,7 @@ def __init__(self):
         self._OmegaPP = PSpecParam("OmegaP", description="Integral of unitless beam power squared over the sky [steradians].", form="(Nbeam_freqs, Npols)", expected_type=np.float64)
         self._beam_freqs = PSpecParam("beam_freqs", description="Frequency bins of the OmegaP and OmegaPP beam-integral arrays [Hz].", form="(Nbeam_freqs,)", expected_type=np.float64)
         self._cosmo = PSpecParam("cosmo", description="Instance of conversion.Cosmo_Conversions class.", expected_type=conversions.Cosmo_Conversions)
-
+        self._store_cov = PSpecParam("store_cov", description="True if storing power spectrum covariance matrix.", expected_type=bool)
```

Comment:
This I think is extraneous: the `_cov_array` can be like all of the other "optional" attributes. If it is not None, then it is considered stored, otherwise it is considered not stored...

Reply:
I've removed the store_cov property.